<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uwu</class>
 <widget class="QMainWindow" name="uwu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>uwu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OpenButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save Button&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>OpenFile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save Button&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>SaveFile</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Description">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>310</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;uwu a text box&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Description&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImageDisplay">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>261</width>
      <height>201</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>261</width>
      <height>221</height>
     </size>
    </property>
    <property name="text">
     <string>         Loaded Imaged Displayed Here</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AuthorName">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Author</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="CopyrightInfo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Copyright Information&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
